marklogic-data-hub/src/test/java/com/marklogic/hub/job/JobManagerTest.java

```diff
@@ -0,0 +1,163 @@
+package com.marklogic.hub.job;
+
+import com.marklogic.client.io.DocumentMetadataHandle;
+import com.marklogic.hub.DataHub;
+import com.marklogic.hub.FlowManager;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.HubTestBase;
+import com.marklogic.hub.flow.*;
+import com.marklogic.hub.scaffold.Scaffolding;
+import org.apache.commons.io.FileUtils;
+import org.custommonkey.xmlunit.XMLUnit;
+import org.junit.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class JobManagerTest extends HubTestBase {
+    private static final String ENTITY = "e2eentity";
+    private static ArrayList<String> jobIds = new ArrayList<String>();
+    private static Path projectDir = Paths.get(".", "ye-olde-project");
+
+    @BeforeClass
+    public static void setupSuite() throws IOException {
+        XMLUnit.setIgnoreWhitespace(true);
+        File projectDirFile = projectDir.toFile();
+        if (projectDirFile.isDirectory() && projectDirFile.exists()) {
```

Comment:
I think there is a method in HubTestBase to delete the project dir for you.

Reply:
Heh. This was a copy/paste. But I'll look for that method.